client/blocks/comments/post-comment-with-error.jsx

```diff
@@ -1,50 +1,51 @@
 import { get } from 'lodash';
 import PropTypes from 'prop-types';
-import { Component } from 'react';
+import { useState } from 'react';
 import PostCommentForm from './form';
 
 import './post-comment.scss'; // intentional, shares styles
 import './post-comment-with-error.scss';
 
-export default class PostCommentWithError extends Component {
-	renderCommentForm() {
-		const { post, commentsTree, commentId, onUpdateCommentText, activeReplyCommentId } = this.props;
-		const commentText = get( commentsTree, [ commentId, 'data', 'content' ] );
-		const commentParentId = get( commentsTree, [ commentId, 'data', 'parent', 'ID' ], null );
-		const placeholderError = get( commentsTree, [ commentId, 'data', 'placeholderError' ] );
-		const placeholderErrorType = get( commentsTree, [ commentId, 'data', 'placeholderErrorType' ] );
+function PostCommentWithError( {
+	activeReplyCommentId,
+	commentId,
+	commentsTree,
+	post,
+	repliesList,
+} ) {
+	const commentParentId = get( commentsTree, [ commentId, 'data', 'parent', 'ID' ], null );
```

Comment:
Now that we're modifying those lines, I wonder if we can use the opportunity to remove those `get()` calls.